src/auth/models/index.ts

```diff
@@ -1 +1,2 @@
-export { Authentication } from './authentication.class'
+export { Strategy } from './strategy.class'
+export { default as strategyFactory } from './strategyFactory.class'
```

Comment:
not a big issue, but any default export becomes the named export anyway, no need to be explicit, e.g.
```
export strategyFactory from './strategyFactory.class'
```

Reply:
Tried it @paddybasi. Not sure it works